x-pack/plugins/security/public/views/management/roles.html

```diff
@@ -1,5 +1,12 @@
-<kbn-management-app section="security">
-  <div class="kuiViewContent kuiViewContent--constrainedWidth kuiViewContentItem">
+<kbn-management-app section="security/roles">
+  <div class="euiPanel euiPanel--paddingLarge euiPageContent" style="flex-grow: 0">
+    <h1 class="euiTitle euiTitle--medium">
+      Roles
+    </h1>
+    <div class="euiText euiTextColor--subdued">
+      <p>Apply roles to groups of users and manage permissions across the stack</p>
```

Comment:
Can we add i18n support for this message?

Reply:
Good catch, forgot to scan the code for these